internal/manager/run.go

```diff
@@ -80,7 +80,10 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic) e
 		retry.Delay(c.KongAdminInitializationRetryDelay),
 		retry.DelayType(retry.FixedDelay),
 		retry.OnRetry(func(n uint, err error) {
-			setupLog.Info("Retrying kong admin api client call #%d/%d after error: %w", n, c.KongAdminInitializationRetries, err)
+			setupLog.Info("Retrying kong admin api client call after error",
+				"#", fmt.Sprintf("%d/%d", n, c.KongAdminInitializationRetries),
```

Comment:
If the log format is json instead of text, then the key would be `"#"` and value `fmt.Sprintf("%d/%d", n, c.KongAdminInitializationRetries)`
Is that what we want?

Reply:
I'm open for suggestions. That was my take on this. If you feel that I should put this back into body then I could that that, np.

Reply:
I would change the key to `initialization-retires`/`InitializationRetries`/`retries` or something alike. Putting that into body works too. The log format can be change via env var, and that would result in some problems with deciphering the meaning.

Btw. it's a limitation of `logursr` middleware (`logrus` wrapper), that the error cannot be properly marshalled into a string. Other, more mature, implementation of `go-logr`, like `go-logr/zapr` (zap wrapper) handle that correctly.